### DISCUSSION TITLE
deploy outline via minio

### DISCUSSION THREAD

**Discussion by RonanCN** [outline/outline#3040] [Self Hosting]
deployed outline( 0.61.0 ) via k8s , and it work now ,but can't upload images via minio. here is the conf.

AWS_REGION: us-east-1
AWS_S3_ACL: private
AWS_S3_FORCE_PATH_STYLE: "true"
AWS_S3_UPLOAD_BUCKET_NAME: outline
AWS_S3_UPLOAD_BUCKET_URL: http://192.168.50.32:9000/outline #the owner-hosted minio address
AWS_S3_UPLOAD_MAX_SIZE: "26214400"
AWS_ACCESS_KEY_ID: my minio-access-key
AWS_SECRET_ACCESS_KEY: my minio-secret-key
MINIO_ACCESS_KEY: my minio-access-key
MINIO_SECRET_KEY: my minio-secret-key

how to fix it ?

**Comment by RonanCN** [comment]
somebody can help ?

**Reply by tommoor** [reply]
You haven't really provided any details on the problem you're seeing… what is the error?

**Comment by wesguirra** [comment]
Having the same problem, cannot upload images, on documents neither on team profile.

**Reply by almereyda** [reply]
Is your issue maybe similar to https://github.com/outline/outline/discussions/3281?

**Comment by sebastian-ruiz** [comment]
Is the bucket URL accessible from the browser JS?
See here:
https://github.com/outline/outline/issues/1100#issuecomment-568555240
